app/models/adviser.rb

```diff
@@ -12,8 +12,6 @@ class Adviser < ActiveRecord::Base
 
   before_validation :upcase_postcode
 
-  after_save :schedule_indexing, if: -> { valid? && geocoded? }
```

Comment:
:heart_eyes_cat: every time an AR filter is deleted, a new pony is born. :heart_eyes_cat: